app/models/low_income_request.rb

```diff
@@ -4,6 +4,8 @@ class LowIncomeRequest < ApplicationRecord
 
   def approve!
     update(status: 'approved')
+    # NOTE: There was a risk here before that, if there were no codes spare then we'd hit an issue
+    LowIncomeCode.create! if LowIncomeCode.available.first.nil?
```

Comment:
If we generate a low-income-code automatically, how do we know the upload it to eventbright? It feels like this pushes this edge case down the road and to a place where it's harder to debug?